spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -79,6 +79,38 @@ object external {
     new File(path).getAbsoluteFile().toURI()
   }
 
+  // ================ HBase entities ======================
+  val HBASE_NAMESPACE_STRING = "hbase_namespace"
+  val HBASE_TABLE_STRING = "hbase_table"
+  val HBASE_COLUMNFAMILY_STRING = "hbase_column_family"
+  val HBASE_COLUMN_STRING = "hbase_column"
+
+  private def stripNameSpace(tableName: String): String = {
+    tableName.substring(tableName.indexOf(":") + 1)
+  }
+
+  def hbaseTbUniqueAttribute(cluster: String, tableName: String, nameSpace: String) =
+    nameSpace.toLowerCase + '.' + stripNameSpace(tableName.toLowerCase()) + '@' + cluster
```

Comment:
Hbase is also case-insensitive, isn't it?

Reply:
Here, we need to make the unique attribute consistent with HBaseAtlasHook: https://github.com/apache/atlas/blob/master/addons/hbase-bridge/src/main/java/org/apache/atlas/hbase/bridge/HBaseAtlasHook.java#L392

Reply:
Then, can we reuse that function here?

Reply:
Then, could you create an issue and PR about upgrading to Atlas 0.8.2 first?

Reply:
yes

Reply:
Resolved.